doc/integrator/ngeo.rst

```diff
@@ -108,3 +108,23 @@ The map configuration will be here:
    If you previously had a CGXP application and want to keep your permalinks
    compatible, the resolutions should be the same as in the previous
    application.
+
+Dynamic.js view
+---------------
+
+To configure the ngeo constants with dynamic or configurable values we have the dynamic view.
+
+This view is configurable in the vars files in the section ``interfaces_config``.
+The sub section is the interface name, and after we have:
+
+* ``redirect_interface``: interface on witch one we can be redirected if we use the wrong type of device
```

Comment:
suggestion:
interface to be redirected to if an unexpected device type is used (mobile/desktop)